lib/rsolr/client.rb

```diff
@@ -63,6 +65,11 @@ def paginate page, per_page, path, opts = nil
     execute build_paginated_request(page, per_page, path, opts)
   end
   
+  # Sends a query to /select or the custom configured select handler.
+  def select opts={}
```

Comment:
The idea is that whatever is using RSolr doesn't need to know which handler to use. The ActiveFedora PR has [this](https://github.com/projecthydra/active_fedora/pull/960/files#diff-737ca785dbb0fa8180646721bf05d662R108) bit. So it's doing a select and using whatever is configured to be the select handler in RSolr. If we didn't have a select method here then ActiveFedora would need to dig into Solr configuration to see which handler needs to be used.

I've taken the approach of having all Solr configuration tied to RSolr. It seemed that the RSolr::Client object is the better place to carry the information about handlers since it's abstracting the connection to Solr. Certainly it would be possible to instead have ActiveFedora know what handler needs to be used and then have it specify that when doing a select or an update.